packages/celotool/src/lib/geth.ts

```diff
@@ -1147,3 +1147,64 @@ export async function connectValidatorPeers(instances: GethInstanceConfig[]) {
     instances.filter(({ wsport, rpcport, validating }) => validating && (wsport || rpcport))
   )
 }
+
+export async function migrateContracts(
+  monorepoRoot: string,
+  validatorPrivateKeys: string[],
+  attestationKeys: string[],
+  validators: string[],
+  to: number = 1000,
+  overrides: any = {},
+  verbose: boolean = true
+) {
+  const migrationOverrides = merge(
```

Comment:
this configuration was devised for e2e test... are we sure it's the overrides we want to launch any geth?

Reply:
well, running a local network is becoming a functionality of the product celotool with this issue: https://github.com/celo-org/celo-monorepo/issues/1807 therefore it makes sense to enable this functionality even outside the e2e tests.

Also i would like to point out this issue: [Proposal] Move e2e tests out of celotool into its own folder https://github.com/celo-org/celo-monorepo/issues/2221 to have a better separation of concerns between celotool and the e2e tests.

Reply:
Ok! My concern was that if somehow we want to have a different config for e2e test, now this is shared with the "local geth", but we can split that later